src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -1139,6 +1139,22 @@ private void ImportCpOpj(int token)
 
         private void ImportUnbox(int token, ILOpcode opCode)
         {
+            TypeDesc type = ResolveTypeToken(token);
+            if (type.IsNullable)
+                throw new NotImplementedException();
+
+            var value = _stack.Pop();
+
+            var boxPtr = LLVM.BuildPointerCast(_builder, value.LLVMValue, LLVM.PointerType(GetLLVMTypeForTypeDesc(type), 0), String.Empty);
```

Comment:
Based on the comment below on boxing, unbox will probably need some adjustments too